kasa/aestransport.py

```diff
@@ -216,7 +214,6 @@ async def perform_login(self):
                     self._default_credentials = get_default_credentials(
                         DEFAULT_CREDENTIALS["TAPO"]
                     )
-                    await asyncio.sleep(self.BACKOFF_SECONDS_AFTER_LOGIN_ERROR)
```

Comment:
Was this only here for the P100, or could we be introducing issues on other devices by removing this?

Reply:
This was only for the P100 and the changes in this PR essentially duplicate the same behaviour to wait https://github.com/python-kasa/python-kasa/pull/751